kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/check/consistency/algorithm/DataMatchDataConsistencyCalculateAlgorithm.java

```diff
@@ -78,7 +78,12 @@ public void init(final Properties props) {
     }
     
     private int getChunkSize(final Properties props) {
-        int result = Integer.parseInt(props.getProperty(CHUNK_SIZE_KEY, DEFAULT_CHUNK_SIZE + ""));
+        int result;
+        try {
+            result = Integer.parseInt(props.getProperty(CHUNK_SIZE_KEY, DEFAULT_CHUNK_SIZE + ""));
+        } catch (final NumberFormatException ex) {
```

Comment:
When a `NumberFormatException` occurs, there is no prompt, and the user has no way of knowing that the configuration is wrong.